common/beerocks/scripts/prplmesh_utils.sh.in

```diff
@@ -104,8 +104,16 @@ status_function() {
     ps -aux | grep local_bus
 }
 
+check_function() {
+    echo "$0: check"
+    if ! grep -q 'CONNECTED --> OPERATIONAL' "/tmp/$USER/beerocks/logs/beerocks_agent.log"; then
+        err "check failed"
+        exit 1
+    fi
+}
+
 usage() {
-    echo "usage: $(basename $0) {start|stop|restart|status} [-hv]"
+    echo "usage: $(basename $0) {start|stop|restart|status|check} [-hv]"
```

Comment:
no need for `check` - should be part of status, return code defines if system is operational, and we can add a --quit flag if we want to get only the check status

Reply:
Requires more changes to status_function, however, because currently it doesn't set the exit code, nor does it use BTL_TYPE or PRPLMESH_MODE to determine which daemons should be running.